Build/docker/mktest.sh

```diff
@@ -0,0 +1,50 @@
+#/bin/bash
+#
+# (C) Copyright IBM Corporation 2017.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ -d /java/jre/bin ];then
+	echo "Using mounted Java8"
+	export JAVA_BIN=/java/jre/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_HOME/bin:$PATH
+	java -version
+elif [ -d /java/bin ]; then
+	echo "Using mounted Java9"
+	export JAVA_BIN=/java/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_HOME/bin:$PATH
+	java -version
+else
+	echo "Using docker image default Java"
+	java_path=$(type -p java)
+	suffix="/java"
+	java_root=${java_path%$suffix}
+	export JAVA_BIN="$java_root"
+	echo "JAVA_BIN is: $JAVA_BIN"
+	$JAVA_BIN/java -version
+fi
+
+export SPEC=linux_x86-64
+export JAVA_VERSION=SE80
+
+
+cd /test/TestConfig
+make -f run_configure.mk
+make getdependency
```

Comment:
This reminds me that we should fold this step into the compile (target depends on target) as we have done elsewhere so it is not needed as an explicit step.

Reply:
yes, thanks for reminding me this. OpenJDK is downloading all dependencies from the get.sh script, and we are going to have the `getdependency` integrated into compile. Change set updated.